app/controllers/TaskController.scala

```diff
@@ -196,15 +196,40 @@ Expects:
       for {
         teams <- taskService.getAllowedTeamsForNextTask(user)
         isTeamManagerOrAdmin <- userService.isTeamManagerOrAdminOfOrg(user, user._organization)
-        (task, initializingAnnotationId) <- taskDAO
+        (taskId, initializingAnnotationId) <- taskDAO
           .assignNext(user._id, teams, isTeamManagerOrAdmin) ?~> "task.unavailable"
-        insertedAnnotationBox <- annotationService.createAnnotationFor(user, task, initializingAnnotationId).futureBox
+        insertedAnnotationBox <- annotationService.createAnnotationFor(user, taskId, initializingAnnotationId).futureBox
         _ <- annotationService.abortInitializedAnnotationOnFailure(initializingAnnotationId, insertedAnnotationBox)
         annotation <- insertedAnnotationBox.toFox
         annotationJSON <- annotationService.publicWrites(annotation, Some(user))
       } yield JsonOk(annotationJSON, Messages("task.assigned"))
     }
   }
+  @ApiOperation(hidden = true, value = "")
+  def assignOne(id: String, userId: String): Action[AnyContent] = sil.SecuredAction.async { implicit request =>
+    log() {
+      for {
+        taskIdValidated <- ObjectId.fromString(id)
+        userIdValidated <- ObjectId.fromString(userId)
+        assignee <- userService.findOneById(userIdValidated, useCache = true)
+        teams <- userService.teamIdsFor(userIdValidated)
+        task <- taskDAO.findOne(taskIdValidated)
+        project <- projectDAO.findOne(task._project)
+        _ <- Fox.assertTrue(userService.isTeamManagerOrAdminOf(request.identity, project._team)) ?~> Messages(
+          "notAllowed")
+        _ <- Fox.assertTrue(userService.isTeamManagerOrAdminOf(request.identity, assignee)) ?~> Messages("notAllowed")
+        (_, initializingAnnotationId) <- taskDAO
+          .assignOneTo(taskIdValidated, userIdValidated, teams) ?~> "task.unavailable"
```

Comment:
Is the error variable missing a `Messages(…)` here?

Reply:
No, the other ones are superfluous, this is added automatically in our Controller traits